packages/schematics/angular/workspace/files/tsconfig.json.template

```diff
@@ -20,9 +20,9 @@
       "es2018",
```

Comment:
I thought we agreed that we'd update the `"module"` to `"es2020"` and `"target"` to "`es2016`" as well.

Did changing these settings break stuff?

Reply:
or is that a separate PR?

Reply:
That will be in a separate PR, which I’ll will be working on in the coming days. Reason being is that changing the module and target is a big change as it changes a lot of files, and I prefer to keep PRs as small as possible, even for reviewers.

Also, the ever green apps changes conflicted with these changes since it touches almost the same files, so I preferred to land that first.

Reply:
Ever green apps PR is finally merged 🥳